packages/formation/js/onDocumentReady.js

```diff
@@ -0,0 +1,10 @@
+import loadAccordionHandler from './accordion';
```

Comment:
@jbalboni Let me know if this is what you were suggesting to do with initializing the accordion.